### PR TITLE
improve MUDROD dataset landing page

### PR DESCRIPTION
**Pull request by quintinali**
1. change terminology of Start/End Date: from $start date (mm/dd/yyyy) - Present (instead of now)
2. change the Format  from RAW to BINARY
3. change units  from ‘m’ to ‘km’
4. language needs to reflect that currently on the PO.DAAC Website
5. change Doi to DOI

### CODE REVIEW

**Comment by quintinali** [comment]
@gxtchen Could you please test these two PRs when you have time?

File: core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java [hunk 1]
```diff
@@ -357,11 +359,11 @@ public String searchByQuery(String index, String type, String query, Boolean bDe
...
           file.addProperty("SpatiallResolution", latResolution + " degrees (latitude) x " + lonResolution + " degrees (longitude)");
         } else {
-
           String acrossResolution = (String) result.get("Dataset-AcrossTrackResolution");
           String alonResolution = (String) result.get("Dataset-AlongTrackResolution");
-
-          file.addProperty("SpatiallResolution", alonResolution + " m (Along) x " + acrossResolution + " m (Across)");
+          double dAcrossResolution = Double.parseDouble(acrossResolution)/1000;
+          double dAlonResolution = Double.parseDouble(alonResolution)/1000;
+          file.addProperty("SpatiallResolution", dAlonResolution + " km (Along) x " + dAcrossResolution + " km (Across)");
```
Comment (lewismc) [inline core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java]: Is ```SpatiallResolution``` spelled correctly?

**Review by lewismc — COMMENTED** [review]
Please see comment @quintinali

File: core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java (hunk 1 above)
Comment (quintinali) [reply core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java]: No, it should be SpatialResolution. I will modify the code, thanks.

**Comment by lewismc** [comment]
+1 please merge into master and update [Ed Armstrong's spreadsheet](https://docs.google.com/spreadsheets/d/16uuq032hYT4s1Cf6ulHX6Y8irGW9xdzBqsw10WSYK0g/edit?usp=sharing) with green rows for whatever has been done. Thank you

**Comment by quintinali** [comment]
@lewismc freg told me to modify UI based on branch 117 (https://github.com/mudrod/mudrod/issues/190),  thus all those modifications are based on branch 117. Do you mean that I should submit a new pull request to the master branch?

**Comment by lewismc** [comment]
@fgreg is correct, the pull request is correct 
```quintinali wants to merge 4 commits into mudrod:mudrod-117 from quintinali:mudrod-190```

**Comment by quintinali** [comment]
oh, I see. You mean I can merge the pull request now. Thanks. I will update related rows in the gdoc.

**Pull request merged**